squirrel_interface_manipulation/src/RPGraspAction.cpp

```diff
@@ -174,16 +435,45 @@ namespace KCL_rosplan {
 				return false;
 			}
 */
+			// Get the object pose.
+			std::stringstream ss;
+			ss << objectID << "_wp";
+			std::vector< boost::shared_ptr<geometry_msgs::PoseStamped> > object_results;
+			if(message_store.queryNamed<geometry_msgs::PoseStamped>(ss.str(), object_results)) {
+				if(results.size()<1) {
+					ROS_ERROR("KCL: (PerceptionAction) aborting waypoint request; no matching object wp %s", ss.str().c_str());
+					publishFeedback(msg->action_id, "action failed");
+					return false;
+				}
+			} else {
+				ROS_ERROR("KCL: (PerceptionAction) could not query message store to fetch object wp %s", ss.str().c_str());
+				publishFeedback(msg->action_id, "action failed");
+				return false;
+			}
+
+			// request manipulation waypoints for object
+			geometry_msgs::PoseStamped &object_wp = *object_results[0];
+
 			// dispatch Grasp action
 			squirrel_manipulation_msgs::BlindGraspGoal goal;
 			goal.heap_center_pose = poseMap;
+			goal.heap_center_pose_static = object_wp;
```

Comment:
error: ‘squirrel_manipulation_msgs::BlindGraspGoal’ has no member named ‘heap_center_pose_static’ 

may a pull request to squirrel_common is missed... (?)

See https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/squirrel_manipulation_msgs/action/BlindGrasp.action